app/views/developers/index.html.erb

```diff
@@ -0,0 +1,108 @@
+</head>
+<body>
+  <div class="container about-container">
+    <div class="app-description">
+    <br>
+      <p>We are the team behind <a href="https://github.com/sciencefixion/glitch_craft">GlitchCraft</a>.
+        GlitchCraft is a Ruby on Rails application allowing users to glitch images.
+         Users can search for images based off a keyword that will return two images
+          for the user's choosing. The first image is created by an AI from scratch
+          based on the keyword via DeepAI, while the second is returned based off
+          search results from human generated image tags in pixabay. </p>
+      <p>The goal of this project was to create a successful Rails application
+        from an idea to delivery within 10 days, focused around solving a
+        problem or making something creative. Learning goals for us were images, file manipulations, storage solutions and fabulous artwork. All of which were achevied!</p>
+      <p> We used an agile process developing our features. Also, Configured a continuous integration server, caching, and implemented a Sinatra Microservice. The tech stack used for this project are the following:</p>
+        <p>Ruby on Rails, Active Record, PostgreSQL, Sinatra, Travis CI, Bootstrap, JavaScript, HTML/CSS, and Trello.</p>
+
+    </div>
+    <div class="center_photo">
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src="https://avatars3.githubusercontent.com/u/37886458?s=460&u=83f7cc97e6e21413954d096159ef462c67513d1e&v=4" alt="Jane Greene" width="600" height="400">
+        </a>
+        <br>
+        <p>Jane Greene</p>
+        <a href="https://www.linkedin.com/in/jane-greene-mba/">LinkedIn</a><a> | </a>
+        <a href="https://github.com/janegreene">GitHub</a>
+      </div>
+
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src= "https://avatars3.githubusercontent.com/u/62079009?s=460&u=ba54507a91cc41193ffa757e661cfc5e708370a7&v=4" alt="Dan Sehl" width="600" height="400">
+        </a>
+        <br>
+        <p>Dan Sehl</p>
+        <a href="https://www.linkedin.com/in/danielsehl/">LinkedIn</a><a> | </a>
+        <a href="https://github.com/dtsehl">GitHub</a>
+      </div>
+
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src="https://avatars1.githubusercontent.com/u/61847658?s=460&u=12c4dff271d79393b5d54867b3d8a73e34286cb1&v=4" alt="Kathy Bui" width="600" height="400">
+        </a>
+        <br>
+        <p>Kathy Bui</p>
+        <a href="https://www.linkedin.com/in/kathy-bui-87a27a1ab/">LinkedIn</a><a> | </a>
+        <a href="https://github.com/Kathybui732">GitHub</a>
+      </div>
+
+      <div class="gallery center-search">
+        <a target="_blank" >
+          <img src="https://avatars2.githubusercontent.com/u/60492885?s=460&u=a23ddc09c48644913428c412632c5535ce5cb703&v=4" alt="Dorion" width="600" height="400">
+        </a>
+        <br>
+        <p>Rion H</p>
```

Comment:
@sciencefixion how do you want your name to look?

Reply:
Just my first name is fine :)